agents-for-bedrock/features-examples/09-fine-grained-access-permissions/README.md

```diff
@@ -0,0 +1,227 @@
+
+```
+$ amplify init
+```
+
+amplify init
+Note: It is recommended to run this command from the root of your app directory
+? Enter a name for the project bedrocksecurity
+The following configuration will be applied:
+
+Project information
+| Name: bedrocksecurity
+| Environment: dev
+| Default editor: Visual Studio Code
+| App type: javascript
+| Javascript framework: react
+| Source Directory Path: src
+| Distribution Directory Path: build
+| Build Command: npm run-script build
+| Start Command: npm run-script start
+
+? Initialize the project with the above configuration? Yes
+Using default provider  awscloudformation
+? Select the authentication method you want to use: AWS profile
+
+For more information on AWS Profiles, see:
+https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html
+
+? Please choose the profile you want to use oktank
+Adding backend environment dev to AWS Amplify app: d5np40989zxhn
+
+Deployment completed.
+Deploying root stack bedrocksecurity [ ---------------------------------------- ] 0/4
+        amplify-bedrocksecurity-dev-1… AWS::CloudFormation::Stack     CREATE_IN_PROGRESS             Sat May 18 2024 15:04:42…     
+        AuthRole                       AWS::IAM::Role                 CREATE_IN_PROGRESS             Sat May 18 2024 15:04:44…     
+        DeploymentBucket               AWS::S3::Bucket                CREATE_IN_PROGRESS             Sat May 18 2024 15:04:44…     
+        UnauthRole                     AWS::IAM::Role                 CREATE_IN_PROGRESS             Sat May 18 2024 15:04:44…     
+
+✔ Help improve Amplify CLI by sharing non-sensitive project configurations on failures (y/N) · no
+
+    You can always opt-in by running "amplify configure --share-project-config-on"
+Deployment state saved successfully.
+✔ Initialized provider successfully.
+✅ Initialized your environment successfully.
+✅ Your project has been successfully initialized and connected to the cloud!
+Some next steps:
+
+"amplify status" will show you what you've added already and if it's locally configured or deployed
+"amplify add <category>" will allow you to add features like user login or a backend API
+"amplify push" will build all your local backend resources and provision it in the cloud
+"amplify console" to open the Amplify Console and view your project status
+"amplify publish" will build all your local backend and frontend resources (if you have hosting category added) and provision it in the cloud
+
+
+```
+$ cp src/aws-exports.tmplt.js  src/aws-exports.js
+```
+
+
+### Step 6.3 - Add Amplify hosting 
+```
+$ amplify add hosting
+```
+
+### Step 6.4 - Publish Amplify project 
+```
+$ amplify publish
+```
+
+When the amplify project is published, it should show a message like this with the url of the frontend application: 
+
+✔ Deployment complete!
+https://code.din67qnkcrl26.amplifyapp.com
```

Comment:
remove link, this should be generated by the application. This one is publicly accessible right now

Reply:
removed it